lib/Jsrt/JsrtSourceHolder.h

```diff
@@ -45,6 +48,7 @@ namespace Js
             scriptLoadCallback(scriptLoadCallback),
             scriptUnloadCallback(scriptUnloadCallback),
             sourceContext(sourceContext),
+            mappedScriptValue(nullptr),
```

Comment:
(add a new comment instead) This line will build break on Full as field not available on NTBUILD.